.github/workflows/publish.yml

```
@@ -52,6 +52,31 @@ jobs:
         env:
           AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
           AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+      -
+        name: Update redirects
+        run: |
+          path_join() {
+            part1=$1
+            part2=$2
+            if [ "${part1: -1}" == "/" ]; then
+              part1=${part1}${part2}
+            else
+              part1=${part1}/${part2}
+            fi
+            echo $part1
+          }
+          for redirect in $(cat ./_site/redirects.json | jq -c '. | to_entries | .[]'); do
+            from=$(echo $redirect | jq '.key')
+            to=$(echo $redirect | jq '.value')
+            from=$(path_join $from "index.html")
```

See above; this could be;

```suggestion
            from="${from%/}/index.html"
```